pkgs/top-level/all-packages.nix

```diff
@@ -31601,6 +31601,8 @@ with pkgs;
 
   fnott = callPackage ../applications/misc/fnott { };
 
+  french-numbers = callPackage ../by-name/fr/french-numbers/package.nix { };
+
```

Comment:
```suggestion
```

Duplicate callPackage

Reply:
Thanks, that wasn't obvious for me from reading by-name's README. I'll propose an update to this file.

Reply:
(done in https://github.com/NixOS/nixpkgs/pull/260022)

Reply:
Btw the plan is to make this an error soon, see https://github.com/NixOS/nixpkgs/pull/256792